### PR TITLE
Created initial guide for Apache Sqoop

### PR DESCRIPTION
**Pull request by iadi7ya**
Apache Sqoop software operators guide.

Created initial documentation guide for Apache Sqoop software operators
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by iadi7ya — COMMENTED** [review]
Created operators guide and connection doc. Modified connection and hooks references. All tests have passed. Thanks

**Review by ryw — APPROVED** [review]
Looks good to me - thank you for contributing documentation @iadi7ya

**Comment by kaxil** [comment]
Can you please rebase your PR on latest Master since we have applied [Black](https://github.com/apache/airflow/commit/4e8f9cc8d02b29c325b8a5a76b4837671bdf5f68) and [PyUpgrade](https://github.com/apache/airflow/commit/8c42cf1b00c90f0d7f11b8a3a455381de8e003c5) on Master.

It will help if your squash your commits into single commit first so that there are less conflicts.

File: docs/howto/connection/sqoop.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
```
Comment (kaxil) [inline docs/howto/connection/sqoop.rst:1]: Can you rebase the PR on latest master please -- we have changed the docs a bit for 2.0.

All the files in this PR need to be moved to https://github.com/apache/airflow/blob/master/docs/apache-airflow-providers-apache-sqoop/index.rst

Comment (kaxil) [reply docs/howto/connection/sqoop.rst:1]: Example: https://github.com/apache/airflow/tree/master/docs/apache-airflow-providers-odbc

**Comment by kaxil** [comment]
@iadi7ya I have updated your other PR https://github.com/apache/airflow/pull/11212/files -- can you fix this one similarly?

**Pull request closed without merging**